apps/client/src/app/pages/tasks/tasks/tasks.component.ts

```diff
@@ -146,7 +146,7 @@ export class TasksComponent {
 
   updateTask(task: LostarkTask): void {
     this.tasksService.updateTask(task);
-  }
+  };
```

Comment:
This `;` is weird, shouldn't be needed, and should trigger a linter error iirc.

Reply:
Fixed it https://github.com/Supamiu/Lostark-helper/pull/56/commits/a2fe1265b0306c98dcc3ba338297835620802a1c ... But any lint error was triggered 🤔